src/main/java/com/gtnewhorizons/angelica/mixins/Mixins.java

```diff
@@ -530,7 +530,13 @@ public enum Mixins {
         .addMixinClasses("mcpatcherforge.ctm_cc.MixinTextureMap")
     ),
     //End from NotFine
-
+//SkidZoom
+    SKIDZOOM(new Builder("zoom zoom")
```

Comment:
please remove all the word "skid" that you put everywhere, it means "stealing code" and that is not appripriate

Reply:
I'm sorry I just seen all these. I'm not super familiar with the way that pull requests work and stuff so I missed all of these my apologies. And as for the word skid, its a copy pasta implementation of a mod I made for myself a while ago that was also called SkidZoom so I apologize for not seeing these messages, that is all. I will turn on notifications for any potential future pull requests.